setup.py

```diff
@@ -6,14 +6,25 @@
 from setuptools import setup, find_packages  # type: ignore
 
 
+def _read_long_description():
+    with open("README.md") as readme:
+        return readme.read()
+
+
 GIT_VERSION = subprocess.check_output("git describe --always".split()).strip().decode("ascii")
-DEV_REQUIRE = ["pytest-cov", "pytest-socket", "tox", "python-semantic-release", "black", "mypy", "pylint", "safety"]
+DEV_REQUIRE = [
+    "pytest-cov", "pytest-socket", "tox", "python-semantic-release", "black", "mypy",
+    "pylint", "safety"
+]
 
 # pylint: disable=line-too-long
 setup(
     name="pytest_rts",
+    description="Coverage-based regression test selection (RTS) plugin for pytest",
+    long_description=_read_long_description(),
+    author="Eero Kauhanen, Matvey Pashkovskiy, Alexey Vyskubov",
     version=GIT_VERSION,
-    packages=find_packages(),
+    packages=find_packages(exclude=("tests",)),
```

Comment:
Would be good to have `tests.*` excluded as well, just as a safety measure for the future.